### DISCUSSION TITLE
Git based CMS for hugo-coder

### DISCUSSION THREAD

**Discussion by bunnis** [luizdepra/hugo-coder#847] [Q&A]
Hello

I'm looking to add a CMS to this theme, however I'm very new in this space. Previously I used this theme in 2021 with the Stackbit and Netlify CMS. Today it seems that everything is different and I want to know what you guys are using. Preferably something that works with github as its repository. ANy ideas?

**Comment by luizdepra** [comment]
Hey!

I don't use any CMS, but I know this one too: [tina.io](https://tina.io).

I think that if the CMS is compatible with Hugo, there will be no problem using hugo-coder.
